docs/github.md

```diff
@@ -112,6 +112,28 @@ $ # Push the updated branch to the remote. Jujutsu automatically makes it a forc
 $ jj git push --branch your-feature
 ```
 
+## Using gh cli
+
+GitHub cli will have trouble finding the proper git repository path in jj repos
+that aren't co-located (see #1008). You can configure the `$GIT_DIR` environment
+variable to point it to the right path:
+
+```shell
+$ GIT_DIR=.jj/repo/store/git gh issue list
+```
+
+You can make that automatic by installing [direnv](https://direnv.net) and
+defining hooks in a .envrc file in the repository root to configure `$GIT_DIR`.
+Just add this line into .envrc:
+
+```shell
+export GIT_DIR=$PWD/.jj/repo/store/git
+```
+
+and run `direnv allow` to approve it for direnv to run. Then GitHub cli will
+work automatically even in co-located repos so you can execute commands like `gh
```

Comment:
`even in co-located repos` wording is weird (wrong?) here since the entire setup leading to that was about making it work in not co-located repos

Reply:
Good catch, fixed.